histomicstk/segmentation/label/passInf.h

```diff
@@ -0,0 +1,5 @@
+#ifndef INFINITY
+
+#define INFINITY 0
+
+#endif
```

Comment:
We probably do not need a separate file for this? 

Can you define it in the isbfcpp.h or the .pyx file itself?